src/core/contrib.js

```diff
@@ -5,7 +5,7 @@
 // #gh-contributors: people whose PR have been merged.
 // Spec editors get filtered out automatically.
 import { flatten, joinAnd } from "./utils";
-import { fetchIndex } from "./github";
+import { getArgs } from "./github-api";
```

Comment:
The name `getArgs` is overly generic so it's not clear what it does... could you give this a better name?